views/layout.hbs

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
You are missing some very important metadatas like the viewport one. Be careful :)